2048/2048/src/main/java/com/appcoins/eskills2048/model/RoomResponseErrorCode.java

```diff
@@ -0,0 +1,6 @@
+package com.appcoins.eskills2048.model;
+
+public enum RoomResponseErrorCode {
+  REGION_NOT_SUPPORTED,
+  DEFAULT_ERROR
```

Comment:
We usually use `GENERIC_ERROR` instead of `DEFAULT_ERROR`

Reply:
changed it!